.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
should we use the fixes via commit: https://help.github.com/articles/closing-issues-via-commit-messages/ instead of listing out separately